.github/ISSUE_TEMPLATE/internal-request.yaml

```diff
@@ -0,0 +1,98 @@
+name: "Documentation request"
+description: Request a documentation change or enhancement for any supported release.
+title: "[REQUEST]: "
+labels: ["request", "Team:Docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this request! This form will create an issue that the Platform Docs team will triage and prioritize.
+
+        ### For best results, complete all fields.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what needs to be documented. What details do users need to know about?
+      placeholder: |
+        What: We're introducing new feature A.
+        When: This feature will launch at the completion of project B.
+        Why: This feature will make X, Y, and Z easier for the user.
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Where can the documentation team learn more about this feature?
+      placeholder: |
+        This feature was implemented in {link_to_PR}.
+
+        This feature was scoped and researched in {link_to_issue}.
+
+        Context for the feature is described in {link_to_internal_doc}.
+    validations:
+      required: true
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set does this change impact?
+      description: Stateful, Serverless, or both?
+      options:
+        - Stateful and Serverless
```

Comment:
>        - Stateful and Serverless

I think per presentation from @florent-leborgne these terms are out-dated (and don't clearly identify the Elastic Cloud Hosted option.  So maybe our options should be something like this?:

> - Elastic On-Prem and Cloud (all)
> - Elastic On-Prem only
> - Elastic Cloud (Hosted and Serverless) only
> - Elastic Cloud Hosted only
> - Elastic Cloud Serverless only
> - Unknown